cluster/gce/util.sh

```diff
@@ -326,16 +326,17 @@ function upload-tars() {
   local node_binary_tar_urls=()
 
   for region in "${PREFERRED_REGION[@]}"; do
+    retention_period="1d"
     suffix="-${region}"
     if [[ "${suffix}" == "-us-central1" ]]; then
-      suffix=""
+      suffix="-us-central1"
```

Comment:
🤔 

what's going on here?

Reply:
why not drop the if statement? why did this exist previously?

Reply:
I honestly don't know why this condition exist. The details in the initial [PR](url) is unclear on why this need to be added.
I'll remove it.